cmd/helm-operator/main.go

```diff
@@ -129,10 +130,11 @@ func main() {
 		os.Exit(0)
 	}
 
-	// TODO(hidde): hacked in so that we can control the Helm version with an
-	// environment variable so that we do not have to make any chart modifications.
-	if ver := getEnvAsSlice("HELM_VERSION", []string{}); len(ver) == 1 {
-		enabledHelmVersions = &ver
+	// Support enabling the Helm supported versions through an
+	// environment variable.
+	helmVersionEnv := getEnvAsSlice("HELM_VERSION", []string{})
```

Comment:
Not part of the PR, but wouldn't it be better to use a flag? The chart could include it too.

Reply:
Also, it raises the question of what should take precedence, the flag or the env variable.

Reply:
A flag always overwrites a set environment variable, that is why there is an explicit check to determine if the default flag value has been changed (`fs.Changed`).

The environment variable was introduced to support setting the value without making any additional adjustments to our Helm chart, and is now being used in some of Stefan his tutorials to only target Helm version 3, which means I can not remove it at the moment.